packages/reporter/src/runnables/runnable-and-suite.tsx

```diff
@@ -86,7 +93,9 @@ class Runnable extends Component<RunnableProps> {
         })}
         data-model-state={model.state}
       >
-        {model.type === 'test' ? <Test model={model as TestModel} /> : <Suite model={model as SuiteModel} />}
+        {model.type === 'test'
+          ? <Test model={model as TestModel} experimentalStudioEnabled={experimentalStudioActive} />
+          : <Suite model={model as SuiteModel} experimentalStudioActive={experimentalStudioActive} />}
```

Comment:
I think we should use `experimentalStudioEnabled` for this prop name. This property describes whether or not Studio is enabled in the Cypress configuration, not whether Studio is currently active. We should be careful not to conflate the two

Reply:
Done, globally renamed to `studioEnabled`. I also dropped the `experimental` prefix, which doesn't add much value internally as pointed out by https://github.com/cypress-io/cypress/pull/23413/files#r951467056